README.md

```diff
@@ -1,4 +1,4 @@
-Styla Widget  2.1.11
+Styla Widget  2.1.12
```

Comment:
I normally only do the version bump once the branch has been merged on stage, and stage is ready to be merged on master. Stage is currently already at 2.1.12.